fileattrs/elfoptimized.attr

```diff
@@ -0,0 +1,4 @@
+%__elf_requires		%{_rpmconfigdir}/elfdeps --requires %{?__filter_GLIBC_PRIVATE:--filter-private}
+%__elf_magic		ELF (32|64)-bit.*$
+%__elf_flags		exeonly
+%__elf_exclude_path     ^(/usr)?/lib(64)?/(i686|x86_64|sse2|haswell|tls)/
```

Comment:
Should be `%__elf_path` instead?

Reply:
Yes. I guess my test is faulty because it seemed to work.